From 05879d5930ceda30b553677f575ad27781ed1581 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@altera.com>
Date: Fri, 31 Jan 2014 11:04:30 -0600
Subject: [PATCH 178/248] FogBugz #177458: Fix the watchdog's parent clock

The watchdog's parent clock is the osc1_clk, and not the per_base_clk.
Correct the error.

[Original patch taken from
git://git.rocketboards.org/linux-socfpga.git socfpga-3.14]

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Dinh Nguyen <dinguyen@altera.com>
---
 arch/arm/boot/dts/socfpga.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/socfpga.dtsi b/arch/arm/boot/dts/socfpga.dtsi
index c601f2f..6823322 100644
--- a/arch/arm/boot/dts/socfpga.dtsi
+++ b/arch/arm/boot/dts/socfpga.dtsi
@@ -852,7 +852,7 @@
 			compatible = "snps,dw-wdt";
 			reg = <0xffd02000 0x1000>;
 			interrupts = <0 171 4>;
-			clocks = <&per_base_clk>;
+			clocks = <&osc1>;
 			status = "disabled";
 		};
 
-- 
1.9.1

